tests/acceptance/CalendarModuleCest.php

```diff
@@ -0,0 +1,31 @@
+<?php 
+
+class CalendarModuleCest
+{
+    public function _before(AcceptanceTester $I)
+    {
+        $I->amOnPage('/index.php');
+        $I->fillField('username', 'admin');
```

Comment:
user/pass should come from test suite config (or somewhere else easy to configure)